client/lib/cgroupslib/mount.go

```diff
@@ -11,15 +11,12 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"syscall"
 
 	"github.com/hashicorp/go-set/v2"
 )
 
 func detect() Mode {
```

Comment:
It's probably time to leave one of those huge documentation block comments describing when things are detected OFF or not.